src/utils/stringUtils.ts

```diff
@@ -0,0 +1,3 @@
+export function cleanId(title: string) {
```

Comment:
Ville kalla den `cleanText` sidan den ikkje er spesifikk for id.

Reply:
Hmm ja, men det blir en ganske stygg tekst som ikke burde brukes til noe synlig. F.eks "Hello. This is test." vil bli "Helo__This_is_test".